TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Management/AssetController.php

```diff
@@ -132,9 +132,9 @@ public function deleteAction(\TYPO3\Media\Domain\Model\Asset $asset)
     {
         try {
             $this->assetService->getRepository($asset)->remove($asset);
-            $this->addFlashMessage('assetHasBeenDeleted', '', Message::SEVERITY_OK, [$asset->getLabel()], 1412375050);
+            $this->addFlashMessage('Asset "%s" has been deleted.', '', Message::SEVERITY_OK, [$asset->getLabel()], 1412375050);
         } catch (AssetServiceException $exception) {
-            $this->addFlashMessage('media.deleteRelatedNodes', '', Message::SEVERITY_WARNING, [], 1412422767);
+            $this->addFlashMessage('Asset could not be deleted, because there are still Nodes using it', '', Message::SEVERITY_WARNING, [], 1412422767);
```

Comment:
Yeah, exactly. There are actually 1-2 translations that belong to Neos but one is already in media package but not in neos and this one the other way around. Didn't looked it up now, i'm on mobiel atm. But the question is how to render it in that partial as we set the Media Package. So guess we should put it in media for now